src/CommandLineUtils/CommandLineApplication.cs

```diff
@@ -597,7 +597,7 @@ public void ShowVersion()
         /// <returns></returns>
         public string GetFullNameAndVersion()
         {
-            return ShortVersionGetter == null ? FullName : string.Format("{0} {1}", FullName, ShortVersionGetter());
+            return ShortVersionGetter == null ? FullName : string.Format("{0} ({1})", FullName, ShortVersionGetter());
```

Comment:
Any particular reason you changed this?

Reply:
Ah sorry.
I wanted align it with the dotnet tool chain.

Reply:
No worries, was just wondering. The change is okay with me.